src/main/java/mate/project/controller/AuthenticationController.java

```diff
@@ -15,9 +18,15 @@
 @RequestMapping("/auth")
 public class AuthenticationController {
     private final UserService userService;
+    private final AuthenticationService authenticationService;
 
     @PostMapping("/registration")
     public UserResponseDto register(@Valid @RequestBody UserRegistrationRequestDto request) {
         return userService.register(request);
     }
+
+    @PostMapping("/login")
+    public UserLoginResponseDto login(@RequestBody UserLoginRequestDto request) {
```

Comment:
You should also add validation, so:
```suggestion
    public UserLoginResponseDto login(@RequestBody @Valid UserLoginRequestDto request) {
```